environments/base/default-config.libsonnet

```diff
@@ -3,6 +3,7 @@
 
   name: 'observatorium-xyz',
   namespace: 'observatorium',
+  s3namespace: 'minio',
```

Comment:
this should be under a `minio` section, and then it can be `namespace` again

Reply:
Done